pubsub/listener.go

```diff
@@ -101,8 +102,8 @@ func (listener *EventsListener) ListenForMessages() {
                     request.GetEvent().GetTransition().GetEvent(), err)))
             continue
         }
-        listener.logger.Info("Event `%s` transitioned FSM [%s] to state `%s` - updating store",
-            request.Event.Transition.Event, smId, fsm.State)
+        listener.logger.Info("Event `%s` transitioned FSM [%s] to state `%s` from state `%s` - updating store",
```

Comment:
please use `Debug` not `Info` - this gets really noisy quickly (I know I used INFO before, my bad)

Reply:
Updated